src/pages/list-view-page.js

```diff
@@ -174,7 +90,97 @@ const PropertiesContainer = styled.div`
 `;
 
 function ListViewPage() {
+  const { filters, setFilters } = useAuth();
   const [properties, setProperties] = useState([]);
+  const [showFilters, setShowFilters] = useState({
+    price: false,
+    propertyType: false,
+    bedBath: false,
+    more:false,
+    operationType:false,
+  });
+  const [prices, setPrices] = useState({
+    min: "",
+    max: ""
+  });
+  const [areas, setAreas] = useState({
+    min: "",
+    max: ""
+  });
+  const [baths, setBaths] = useState(1);
+  const [beds, setBeds] = useState(1);
+  const [petAllowed, setPetAllowed] = useState(false);
+  const [apartment, setApartment] = useState(false);
+  const [house, setHouse] = useState(false);
+  const [rent, setRent] = useState(false);
+  const [buy, setBuy] = useState(false);
+  const [both, setBoth] = useState(false);
+  const [query, setQuery] = useState("");
```

Comment:
Consider some future enhancements this using `React.useReducer` or an object in one `useState`